SMLHelper/MonoBehaviours/EntitySpawner.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using SMLHelper.V2.Handlers;
+using UnityEngine;
+using UWE;
+
+namespace SMLHelper.V2.MonoBehaviours
+{
+    public class EntitySpawner : MonoBehaviour
+    {
+        public SpawnInfo spawnInfo;
+
+        void Start() => StartCoroutine(SpawnAsync());
+
+        IEnumerator SpawnAsync()
+        {
+            var task = new TaskResult<GameObject>();
+            yield return GetPrefabAsync(task);
+
+            var obj = UWE.Utils.InstantiateDeactivated(task.Get(), spawnInfo.spawnPosition, Quaternion.identity);
+            
```

Comment:
Add a null check with error logging on the result just incase the get prefab fails to get a prefab